### ISSUE TITLE
Accessibility: Merge Sections Screenreader menu

### ISSUE THREAD

**Issue by ktowneUM** [tl-its-umich-edu/canvas-course-manager-next#640]
Based on the [CCM Accessibility Review](https://docs.google.com/spreadsheets/d/1mYrbqwFLLXQoGRTY_XEGkfS3cu3BO_pUN_zQ_DFdhUY/edit?gid=1709548063#gid=1709548063):

Severity: Critical

Feature: Merge Sections

Issue notes:
NVDA announced "Menu button submenu sort" if this button does not open a menu or a popup, "aria-haspopup="true" and aria-controls="simple-menu" should be removed from the button. Select All button is Okay.

Criterion:
